BUILD.bazel

```diff
@@ -45,6 +45,34 @@ java_binary(
     ],
 )
 
+java_binary(
```

Comment:
Nit: Can we add a summary comment above these two rules, which also indicate these are debugging tools?